content/docs/sidebar.json

```diff
@@ -217,7 +217,21 @@
       {
         "label": "experiments",
         "slug": "exp",
-        "source": "exp/index.md"
+        "source": "exp/index.md",
+        "children": [
+          {
+            "label": "exp run",
+            "slug": "run"
+          },
```

Comment:
Hiii @rogermparent another quick callout... Would it be possible to automatically label subcommands inside `children` as `parent children` to avoid all these `{}` structures (here and in other subcommand groups). Ty

Reply:
Something like replacing `{ "label": "exp run", "slug": "run" }` with the string `"run"` having the same effect? That's a good idea, and totally doable!

Reply:
Great! I was just thinking to replace the whole `{}` block with `"run"` in this case (:

Reply:
Haha yeah Roger you got it exactly right the first time. Didn't read your answer correctly 😬

Reply:
Any hope we can get that done for this PR @rogermparent? No worries if not but let's make make an issue then? Thanks